packages/bpk-component-text/src/BpkText-test.js

```diff
@@ -63,6 +63,21 @@ describe('BpkText', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('should render correctly with tageName="text", inside an svg', () => {
```

Comment:
For this test can we remove the svg parts and just test that the tagName `text` works as an input prop, as this will applicable for any use and not just in SVGs